### PR TITLE
fix: CRM dashboard fixes

### PR DESCRIPTION
**Pull request by anupamvs**
**Issue**

- Overlapping of Legends
- Empty Incoming Lead Chart

**Fix**
- Adding truncateLegends and maxSlices for Donut and Pie chart
image

### CODE REVIEW

File: erpnext/crm/dashboard_fixtures.py [hunk 1]
```diff
@@ -90,7 +90,11 @@ def get_charts():
         'timeseries': 1,
         "owner": "Administrator",
         "filters_json": json.dumps([["Opportunity", "company", "=", company, False]]),
-        "type": "Pie"
+        "type": "Pie",
+        "custom_options": json.dumps({
+            "truncateLegends": 1,
+            "maxSlices": 5
```
Comment (nabinhait) [inline erpnext/crm/dashboard_fixtures.py]: 5 is little less, show 8

File: erpnext/crm/dashboard_fixtures.py [hunk 2]
```diff
@@ -123,7 +127,11 @@ def get_charts():
             ["Opportunity", "company", "=", company, False]
         ]),
         "owner": "Administrator",
-        "type": "Donut"
+        "type": "Donut",
+        "custom_options": json.dumps({
+            "truncateLegends": 1,
+            "maxSlices": 5
```
Comment (nabinhait) [inline erpnext/crm/dashboard_fixtures.py]: 5 is little less, show 8

File: erpnext/crm/dashboard_fixtures.py [hunk 3]
```diff
@@ -152,7 +160,11 @@ def get_charts():
         "document_type": "Lead",
         'is_public': 1,
         "owner": "Administrator",
-        "type": "Pie"
+        "type": "Pie",
+        "custom_options": json.dumps({
+            "truncateLegends": 1,
+            "maxSlices": 5
```
Comment (nabinhait) [inline erpnext/crm/dashboard_fixtures.py]: 5 is little less, show 8

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/crm/dashboard_fixtures.py (hunk 1 above)
Comment (anupamvs) [reply erpnext/crm/dashboard_fixtures.py]: Done!

File: erpnext/crm/dashboard_fixtures.py (hunk 2 above)
Comment (anupamvs) [reply erpnext/crm/dashboard_fixtures.py]: Done

File: erpnext/crm/dashboard_fixtures.py (hunk 3 above)
Comment (anupamvs) [reply erpnext/crm/dashboard_fixtures.py]: Done!

**Review by nabinhait — APPROVED** [review]

**Comment by nabinhait** [comment]
@Mergifyio backport version-13-beta-pre-release

**Pull request merged**